### ISSUE TITLE
Create Function Comment doesn't work with this function

### ISSUE THREAD

**Issue by mishaxz** [L-I-V/MQL-Tools#3]
```cpp
void closeall(string symbol="", ushort maxSlippage=0, int magic=0, bool todays=false) {

}

double openProfit(const string symbol="", bool netProfit=false, int magic=0, todays=false) {
return 0;
}
```

**Comment by mishaxz** [comment]
it works in general, with other functions, I think it has problem parsing these parameters

**Comment by L-I-V** [comment]
I will fix it in the near future

**Comment by mishaxz** [comment]
thanks

**Comment by L-I-V** [comment]
I fixed it, in version 0.1.1

**Comment by mishaxz** [comment]
great, thanks
